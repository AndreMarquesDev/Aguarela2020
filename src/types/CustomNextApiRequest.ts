import type { NextApiRequest } from 'next';
import type { FormPostRequestBody } from './FormPostRequestBody';

export type CustomNextApiRequest = {
    body: FormPostRequestBody;
} & NextApiRequest;
