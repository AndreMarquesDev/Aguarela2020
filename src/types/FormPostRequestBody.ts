import type { FormValues } from './FormValues';

export type FormPostRequestBody = {
    isPlaywright: boolean;
} & FormValues;
