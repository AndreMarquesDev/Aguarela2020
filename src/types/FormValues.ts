export type FormValues = {
    name: string;
    email: string;
    message: string;
    brand: string;
    subject: string;
};
