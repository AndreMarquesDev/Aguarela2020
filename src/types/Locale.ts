export enum Locale {
    Pt = 'pt',
    En = 'en',
}
