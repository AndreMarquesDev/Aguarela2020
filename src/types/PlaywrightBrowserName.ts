export enum PlaywrightBrowserName {
    Chromium = 'chromium',
    Firefox = 'firefox',
    Webkit = 'webkit',
}
