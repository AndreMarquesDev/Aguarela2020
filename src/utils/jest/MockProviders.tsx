/* eslint-disable react/jsx-no-constructed-context-values */
import React, { ReactNode } from 'react';
import { NavLinksContext } from '../../components/context/NavLinksContext';
import { TextsContext } from '../../components/context/TextsContext';
import { Locale } from '../../types/Locale';
import { textsEn, textsPt } from '../texts';

interface MockProvidersProps {
    language?: Locale;
    isMenuOpen?: boolean;
    toggleMenu?: jest.Mock;
    setNavHeight?: jest.Mock;
    children: ReactNode;
}

export const MockProviders = ({
    language = Locale.Pt,
    isMenuOpen = false,
    toggleMenu = jest.fn(),
    setNavHeight = undefined,
    children,
}: MockProvidersProps): JSX.Element => {
    return (
        <TextsContext.Provider
            value={{
                texts: language === Locale.Pt ? textsPt : textsEn,
            }}
        >
            <NavLinksContext.Provider value={{ isMenuOpen, toggleMenu, setNavHeight }}>
                {children}
            </NavLinksContext.Provider>
        </TextsContext.Provider>
    );
};
