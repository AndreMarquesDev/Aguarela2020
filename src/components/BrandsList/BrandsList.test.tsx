import type { RenderResult } from '@testing-library/react';
import { render, screen } from '@testing-library/react';
import React from 'react';
import { Locale } from '../../types/Locale';
import { MockProviders } from '../../utils/jest/MockProviders';
import { textsEn, textsPt } from '../../utils/texts';
import { BrandsList } from './BrandsList';
import '@testing-library/jest-dom';

const renderComponent = (language: Locale = Locale.Pt): RenderResult => {
    return render(
        <MockProviders language={language}>
            <BrandsList />
        </MockProviders>,
    );
};

describe('<BrandsList />', () => {
    it('renders properly', () => {
        const { container } = renderComponent();

        expect(screen.getByText(textsPt.myNetwork)).toBeInTheDocument();

        expect(container).toMatchSnapshot();
    });

    it('renders properly in English', () => {
        const { container } = renderComponent(Locale.En);

        expect(screen.getByText(textsEn.myNetwork)).toBeInTheDocument();

        expect(container).toMatchSnapshot();
    });
});
