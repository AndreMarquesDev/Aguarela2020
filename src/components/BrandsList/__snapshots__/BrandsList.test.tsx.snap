// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrandsList /> renders properly 1`] = `
<div>
  <section
    class="jsx-3152002307 container"
    data-testid="brandsList_section"
  >
    <div
      class="jsx-3152002307 wrapper genericMargins"
    >
      <h1
        class="jsx-1254015016 fontL colored"
      >
        A minha rede
      </h1>
      <ul
        class="jsx-3152002307"
      >
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="avocado"
            class="jsx-3152002307"
            href="https://www.instagram.com/avocadohouselisbon/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="avocado logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="pernod"
            class="jsx-3152002307"
            href="https://www.pernod-ricard.com"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="pernod logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="tjela"
            class="jsx-3152002307"
            href="https://www.instagram.com/tudonatjela/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="tjela logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="guacamole"
            class="jsx-3152002307"
            href="https://www.instagram.com/guacamolegmg/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="guacamole logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="kaffeehaus"
            class="jsx-3152002307"
            href="https://www.instagram.com/kaffeehaus_lisboa/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="kaffeehaus logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="taste of india"
            class="jsx-3152002307"
            href="https://www.instagram.com/tasteofindiapt/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="taste of india logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="icecream roll"
            class="jsx-3152002307"
            href="https://www.instagram.com/icecreamroll.pt/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="icecream roll logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="marialimao"
            class="jsx-3152002307"
            href="https://www.instagram.com/bebemarialimao/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="marialimao logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="biergarten"
            class="jsx-3152002307"
            href="https://www.instagram.com/biergarten.jardimdacerveja/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="biergarten logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="trattoria"
            class="jsx-3152002307"
            href="https://www.instagram.com/trattoriadipaola/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="trattoria logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="a amiga esteticista"
            class="jsx-3152002307"
            href="https://www.instagram.com/aamigaesteticista/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="a amiga esteticista logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="rice me deli"
            class="jsx-3152002307"
            href="https://www.instagram.com/ricemedeli/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="rice me deli logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="4 patas de 5 estrelas"
            class="jsx-3152002307"
            href="https://www.instagram.com/4patasde5estrelas/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="4 patas de 5 estrelas logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="rice me"
            class="jsx-3152002307"
            href="https://www.instagram.com/ricemerestaurante/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="rice me logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="luminous"
            class="jsx-3152002307"
            href="https://www.instagram.com/becomeluminouscoaching/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="luminous logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="mad mary"
            class="jsx-3152002307"
            href="https://www.instagram.com/madmarycuisine/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="mad mary logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="harpoon"
            class="jsx-3152002307"
            href="https://www.linkedin.com/company/harpoon-lda"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="harpoon logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="bovine"
            class="jsx-3152002307"
            href="https://www.instagram.com/bovinelisboa/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="bovine logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
      </ul>
    </div>
  </section>
</div>
`;

exports[`<BrandsList /> renders properly in English 1`] = `
<div>
  <section
    class="jsx-3152002307 container"
    data-testid="brandsList_section"
  >
    <div
      class="jsx-3152002307 wrapper genericMargins"
    >
      <h1
        class="jsx-1254015016 fontL colored"
      >
        My network
      </h1>
      <ul
        class="jsx-3152002307"
      >
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="avocado"
            class="jsx-3152002307"
            href="https://www.instagram.com/avocadohouselisbon/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="avocado logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Favocado.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="pernod"
            class="jsx-3152002307"
            href="https://www.pernod-ricard.com"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="pernod logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fpernod.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="tjela"
            class="jsx-3152002307"
            href="https://www.instagram.com/tudonatjela/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="tjela logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftjela.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="guacamole"
            class="jsx-3152002307"
            href="https://www.instagram.com/guacamolegmg/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="guacamole logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fguacamole.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="kaffeehaus"
            class="jsx-3152002307"
            href="https://www.instagram.com/kaffeehaus_lisboa/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="kaffeehaus logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fkaffeehaus.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="taste of india"
            class="jsx-3152002307"
            href="https://www.instagram.com/tasteofindiapt/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="taste of india logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftasteofindia.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="icecream roll"
            class="jsx-3152002307"
            href="https://www.instagram.com/icecreamroll.pt/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="icecream roll logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ficecreamroll.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="marialimao"
            class="jsx-3152002307"
            href="https://www.instagram.com/bebemarialimao/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="marialimao logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fmarialimao.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="biergarten"
            class="jsx-3152002307"
            href="https://www.instagram.com/biergarten.jardimdacerveja/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="biergarten logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fbiergarten.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="trattoria"
            class="jsx-3152002307"
            href="https://www.instagram.com/trattoriadipaola/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="trattoria logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Ftrattoria.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="a amiga esteticista"
            class="jsx-3152002307"
            href="https://www.instagram.com/aamigaesteticista/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="a amiga esteticista logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Famiga-esteticista.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="rice me deli"
            class="jsx-3152002307"
            href="https://www.instagram.com/ricemedeli/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="rice me deli logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fricemedeli.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="4 patas de 5 estrelas"
            class="jsx-3152002307"
            href="https://www.instagram.com/4patasde5estrelas/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="4 patas de 5 estrelas logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2F4patas.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="rice me"
            class="jsx-3152002307"
            href="https://www.instagram.com/ricemerestaurante/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="rice me logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Friceme.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="luminous"
            class="jsx-3152002307"
            href="https://www.instagram.com/becomeluminouscoaching/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="luminous logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fluminous.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="mad mary"
            class="jsx-3152002307"
            href="https://www.instagram.com/madmarycuisine/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="mad mary logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fmadmary.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="harpoon"
            class="jsx-3152002307"
            href="https://www.linkedin.com/company/harpoon-lda"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="harpoon logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fharpoon.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
        <li
          class="jsx-3152002307"
        >
          <a
            aria-label="bovine"
            class="jsx-3152002307"
            href="https://www.instagram.com/bovinelisboa/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="bovine logo"
              data-nimg="1"
              decoding="async"
              height="150"
              loading="lazy"
              src="/_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=384&q=75"
              srcset="/_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FbrandLogos%2Fbovine.jpg&w=384&q=75 2x"
              style="color: transparent; width: 100%; height: auto; position: relative; max-width: 150px;"
              width="150"
            />
          </a>
        </li>
      </ul>
    </div>
  </section>
</div>
`;
